### PR TITLE
designate: deploy designate on crowbar jobs by default(SOC-10288)

### PR DESCRIPTION
**Pull request by gosipyan**
- designate: deploy designate on crowbar jobs by default(SOC-10288)
- Apply designate-worker in mkcloud configuration to cluster setup

### CODE REVIEW

**Comment by rsalevsky** [comment]
Can we do the same for mkcloud based jobs?

File: jenkins/ci.suse.de/templates/cloud-crowbar-pipeline-template.yaml [hunk 1]
```diff
@@ -170,9 +170,9 @@
           multi-select-delimiter: ','
           # NOTE: keeping designate and octavia disabled by default, until their
           # implementation is complete
-          default-value: '{enabled_services|database,rabbitmq,keystone,swift,monasca,glance,cinder,neutron,nova,horizon,ceilometer,heat,manila,trove,barbican,magnum,sahara,aodh,tempest}'
+          default-value: '{enabled_services|designate,database,rabbitmq,keystone,swift,monasca,glance,cinder,neutron,nova,horizon,ceilometer,heat,manila,trove,barbican,magnum,sahara,aodh,tempest}'
           value: >-
-            database,rabbitmq,keystone,swift,monasca,glance,cinder,neutron,nova,
+            designate,database,rabbitmq,keystone,swift,monasca,glance,cinder,neutron,nova,
```
Comment (stefannica) [inline jenkins/ci.suse.de/templates/cloud-crowbar-pipeline-template.yaml]: you already have designate in the list of possible values

File: jenkins/ci.suse.de/templates/cloud-crowbar-pipeline-template.yaml [hunk 2]
```diff
@@ -170,9 +170,9 @@
           multi-select-delimiter: ','
           # NOTE: keeping designate and octavia disabled by default, until their
           # implementation is complete
-          default-value: '{enabled_services|database,rabbitmq,keystone,swift,monasca,glance,cinder,neutron,nova,horizon,ceilometer,heat,manila,trove,barbican,magnum,sahara,aodh,tempest}'
+          default-value: '{enabled_services|designate,database,rabbitmq,keystone,swift,monasca,glance,cinder,neutron,nova,horizon,ceilometer,heat,manila,trove,barbican,magnum,sahara,aodh,tempest}'
```
Comment (stefannica) [inline jenkins/ci.suse.de/templates/cloud-crowbar-pipeline-template.yaml]: enabling designate by default will impact all Crowbar ECP jobs. we shouldn't do it unless we know for a fact that it deploys successfully and doesn't cause any problems at runtime

Comment (stefannica) [reply jenkins/ci.suse.de/templates/cloud-crowbar-pipeline-template.yaml]: ... and that includes running tempest

Comment (gosipyan) [reply jenkins/ci.suse.de/templates/cloud-crowbar-pipeline-template.yaml]: this PR can be merged after https://github.com/crowbar/crowbar-openstack/pull/2282 and https://github.com/crowbar/crowbar-openstack/pull/2294 merged .

File: jenkins/ci.suse.de/templates/cloud-crowbar-pipeline-template.yaml [hunk 3]
```diff
@@ -170,7 +170,7 @@
           multi-select-delimiter: ','
           # NOTE: keeping designate and octavia disabled by default, until their
```
Comment (flaviodsr) [inline jenkins/ci.suse.de/templates/cloud-crowbar-pipeline-template.yaml]: Could you remove that comment regarding designate?

File: jenkins/ci.suse.de/templates/cloud-crowbar-pipeline-template.yaml [hunk 4]
```diff
@@ -170,7 +170,7 @@
...
           # implementation is complete
-          default-value: '{enabled_services|database,rabbitmq,keystone,swift,monasca,glance,cinder,neutron,nova,horizon,ceilometer,heat,manila,trove,barbican,magnum,sahara,aodh,tempest}'
+          default-value: '{enabled_services|designate,database,rabbitmq,keystone,swift,monasca,glance,cinder,neutron,nova,horizon,ceilometer,heat,manila,trove,barbican,magnum,sahara,aodh,tempest}'
```
Comment (flaviodsr) [inline jenkins/ci.suse.de/templates/cloud-crowbar-pipeline-template.yaml]: Would be nice if you could keep the same ordering as the `value` parameter.

Comment (gosipyan) [reply jenkins/ci.suse.de/templates/cloud-crowbar-pipeline-template.yaml]: fixed

**Pull request merged**